docs/src/main/asciidoc/infinispan-client.adoc

```diff
@@ -159,21 +222,25 @@ Here is an example of how the preceding classes should be changed:
 If your classes have only mutable fields, then the `ProtoFactory` annotation
 is not required, assuming your class has a no arg constructor.
 
-Then all that is required is a very simple `SerializationContextInitializer` interface with an annotation
+Then all that is required is a very simple `GeneratedSchema` interface with an annotation
 on it to specify configuration settings
 
-.BookContextInitializer.java
+.BooksSchema.java
 [source,java]
 ----
-import org.infinispan.protostream.SerializationContextInitializer;
+import org.infinispan.protostream.GeneratedSchema;
 import org.infinispan.protostream.annotations.AutoProtoSchemaBuilder;
 import org.infinispan.protostream.types.java.math.BigDecimalAdapter;
 
 @AutoProtoSchemaBuilder(includeClasses = { Book.class, Author.class, BigDecimalAdapter.class }, schemaPackageName = "book_sample")
-interface BookContextInitializer extends SerializationContextInitializer {
+interface BookStoreSchema extends GeneratedSchema {
 }
 ----
 
+[TIP]
+Protostream provides default Protobuf mappers for commonly used types as `BigDecimal`,
+included under the `org.infinispan.protostream.types` package.
```

Comment:
The protostream module would need to be added to the project, it isn't a package in the normal protostream module.

Reply:
I would like to include the dependency in the client. I thought it was since I use it in the integration tests but indeed it's not in the client